text/0000-tuple-struct-self-ctor.md

```diff
@@ -139,23 +224,116 @@ should work and will probably try at some point.
 # Reference-level explanation
 [reference-level-explanation]: #reference-level-explanation
 
-Rust (now) allows usage of `Self(v0, v1, ..)` inside inherent
-and trait `impl`s of tuple structs, either when mentioning the
-tuple struct directly in the `impl` header, or via a type alias.
+Rust (now) allows usage of `Self(v0, v1, ..)` inside inherent and
```

Comment:
I find this `now` confusing. Does it mean before or after the RFC?

Reply:
After the RFC :)